youtube_dl/YoutubeDL.py

```diff
@@ -334,6 +334,11 @@ class YoutubeDL(object):
         'playlist_index',
     ))
 
+    _NUMERIC_FIELDS |= set(('longside', 'shortside',))
```

Comment:
If we're doing this, I'd just have these on a separate line in the original list:
```py
        'width', 'height', 'tbr', 'abr', 'asr', 'vbr', 'fps', 'filesize', 'filesize_approx',
        'longside', 'shortside',
        'timestamp', 'upload_year', 'upload_month', 'upload_day',
```

Reply:
As suggested in my refactor idea, IMO this list should not exist in the first place; rather, its values should be collected from the sections in code that implement them. I will not oppose anyone who decides to enact your solution, but I won't do anything that may look like approval.